tests/x11/addon_products_via_SCC_yast2.pm

```diff
@@ -0,0 +1,67 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: add addon to SLES via SCC
+# Maintainer: QE YaST <qa-sle-yast@suse.de>
+
+use base qw(y2_installbase y2_module_guitest);
+use strict;
+use warnings;
+use testapi;
+use registration;
+use version_utils qw(is_sle is_leap);
+use x11utils 'turn_off_gnome_screensaver';
+use YaST::Module;
+use utils;
+
+sub test_setup {
+    select_console "x11";
+    x11_start_program('xterm');
+    turn_off_gnome_screensaver;
+}
+
+sub run {
+    my $self = shift;
+    test_setup;
+    YaST::Module::open(module => 'scc', ui => 'qt');
+    save_screenshot;
+    $testapi::distri->get_registration()->register_via_scc({
+            email => get_var('SCC_EMAIL'),
+            reg_code => get_var('SCC_REGCODE')});
+    save_screenshot;
+    my @scc_addons = split ',', get_var('SCC_ADDONS');
+    $testapi::distri->get_module_registration()->register_extension_and_modules([@scc_addons]);
+    save_screenshot;
```

Comment:
could you please separate with a blank line the actions in the test like in here? it makes the reading easier and in the future we might find a way to integrate the screenshot better with the function itself, not like a separate line. So for example, one blank line after opening the module, another one after each screenshot or action that lead you to other screen, so visually it is easy to match the code with the visualization (feel free to ignore this comment, it is just a suggestion that is just cosmetic).